MyClass.java

```diff
@@ -3,7 +3,18 @@
 
 public class MyClass {
 	public static void main(String args[]) {
-	   ProgramIntro();
+		// Subteam key
+		final int BUILD = 0;
+		final int DESIGN = 1;
+		final int FINANCE = 2;
+		final int GRAPHICS = 3;
+		final int MARKETING = 4;
+		final int MEDIA = 5;
+		final int OUTREACH = 6;
+		final int PROGRAMMING = 7;
+		final int INITIATIVES = 8;
```

Comment:
Love the keys. Java has a concept called enums that replicate/is used to solve what you are doing here.

https://www.w3schools.com/java/java_enums.asp

I'd suggest using it instead.

Also since these are finals and may want to be used in other functions, it would be better served in the `MyClass` scope instead of the `main()` function scope. i.e. - Move these declarations above the 

```
public static void main(String args[])
```

line.

Reply:
Looking at this a little closer. Was the plan to use these later? I'm not seeing them used in the code you've added.

Reply:
Yeah the plan was to simplify the process for whoever ends up applying the data, so they could use the keys in the array the method returns to access the value, without having to focus too much attention to the format of the array returned. They aren't used within the calculation method.

Reply:
But I will definitely look at the changes suggested